packages/next/shared/lib/loadable.js

```diff
@@ -66,13 +66,16 @@ function createLoadableComponent(loadFn, options) {
       timeout: null,
       webpack: null,
       modules: null,
+      suspense: false,
     },
     options
   )
 
   let subscription = null
-
   function init() {
+    if (opts.suspense) {
+      return Promise.resolve()
+    }
```

Comment:
I think it's probably OK to remove the `preload` method when using Suspense for now, so `init` should never be called.